mobot/mobot_client/item_drop_session.py

```diff
@@ -0,0 +1,563 @@
+# Copyright (c) 2021 MobileCoin. All rights reserved.
+
+import os
+import full_service_cli as mc
+import googlemaps
+import enum
+import pytz
+
+from mobot_client.drop_session import BaseDropSession, ItemSessionState
+from mobot_client.models import (
+    DropSession,
+    CustomerStorePreferences,
+    Order,
+    Sku,
+    CustomerDropRefunds,
+)
+from mobot_client.chat_strings import ChatStrings
+
+
+class OrderStatus(enum.Enum):
+    STARTED = 0
+    CONFIRMED = 1
+    SHIPPED = 2
+    CANCELLED = 3
+
+
+class ItemDropSession(BaseDropSession):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        gmaps_client_key = os.environ["GMAPS_CLIENT_KEY"]
+        self.gmaps = googlemaps.Client(key=gmaps_client_key)
+        
+        self.vat_id = os.environ["VAT_ID"]
+
+    @staticmethod
+    def drop_item_has_stock_remaining(drop):
+        # FIXME: Unused
+        skus = Sku.objects.fliter(item=drop.item)
+        for sku in skus:
+            number_ordered = Order.objects.filter(sku=sku).exclude(status=OrderStatus.CANCELLED.value).count()
+            if number_ordered < sku.quantity:
+                return True
+
+        return False
+
+    def drop_item_get_available(self, drop_item):
+        available_options = []
+        skus = Sku.objects.filter(item=drop_item).order_by("sort_order")
```

Comment:
Same, we should be using ForeignKey's built-in magic that puts an "skus" attribute on the drop_item object if there's a ForeignKey field referencing it on the `Sku` object.

Reply:
@sugargoat This change is already in my branch, so don't worry about it just yet.